lottie/src/main/java/com/airbnb/lottie/network/LottieNetworkFetcher.java

```diff
@@ -12,4 +12,6 @@
     @WorkerThread
     @NonNull
     LottieNetworkResult fetchSync(@NonNull String url) throws IOException;
+
+    void disconnect();
```

Comment:
Can you provide a javadoc for people who want to use this interface? When and why would you call disconnect?
Could you also add a more complete javadoc for the class? Something like:
```
Implement this interface to handle network fetching manually when animations are requested via url. By default, Lottie will use an HttpUrlConnection under the hood but this enables you to hook into your own network stack. By default, Lottie will also handle caching the animations but if you want to provide your own cache directory, you may implement `LottieNetworkCacheProvider`.

@see Lottie#initialize
```

Reply:
done